cli/flox/src/commands/init/mod.rs

```diff
@@ -79,6 +80,7 @@ impl Init {
             EnvironmentName::from_str(DEFAULT_NAME)?
         } else {
             let name = dir
+                .parse_dot()?
```

Comment:
nb: while this mitigates the error we're seeing, we could normalize the path earlier either for this method (up in L74) or at the argument parsing level.

Since relative paths have yet to cause issues outside of this context we may as well keep it local here.

Reply:
It also makes sense to use and report the original path in any error messages so that it matches the user's CLI arguments.